compatibility/bazel_tools/testing.bzl

```diff
@@ -427,7 +439,7 @@ def sdk_platform_test(sdk_version, platform_version):
             dar_files = dar_files,
         )],
         tags = ["exclusive", sdk_version, platform_version] + extra_tags(sdk_version, platform_version),
-    )
+    ) if ledger_api_test_tool_compatible(sdk_version, platform_version) else None
```

Comment:
is there a reason why this does not go via the `exclusions` matrix?

Reply:
Mostly because it felt a bit weird to run the tests but excluding _everything_.

Reply:
oh right, fair point. Maybe we should have a special field for that in the test matrix.